app/styles/components/_users-pagination-links.scss

```diff
@@ -0,0 +1,14 @@
+.users-pagination {
```

Comment:
Yea. That's one if the reasons I wanted this fixed. That way it will be easer to straighten out when we go to angle bracket components.